### PR TITLE
Spilts pmcs from nt lorewise, and gives pmc gunner the modern smartgun

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
title basically, spilts pmcs from NT lorewise as in my opinion it doesn't really make much sense that NT has their own private military contracting group.
Also updates pmcs smartgun to use the modern smartgun rather than the old one, gives them the combat lightpack to carry drums.

## Why It's Good For The Game
Updated smartgun should've been done ages ago.
the PMCs being separate from NT organizationally makes more sense to me. But I can revert it if nobody wants it.

## Changelog
:cl:
tweak: NT pmcs are no longer part of NT, and are only hired by them.
balance: pmc sgs get the smartmachinegun
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -64. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by MrDoomBringer** [comment]
1. nanotrasen is a mega corperation it makes total sense for them to have / pay a PMC isn't that the whole point of them
2. delta of -64 what da

**Comment by Pariah919** [comment]
>nanotrasen is a mega corperation it makes total sense for them to have / pay a PMC isn't that the whole point of a pmc
>
they're hired, not part of nt

**Comment by MLGTASTICa** [comment]
NT is a megacorporation , being hired equals to being part to me.

**Review by Hughgent — APPROVED** [review]

File: code/datums/emergency_calls/pmc.dm [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 
 /datum/emergency_call/pmc/print_backstory(mob/living/carbon/human/H)
-	to_chat(H, "<B>After leaving your [pick(75;"distant", 20;"close", 5;"ever-lovingly close")] [pick("family", "friends", "band of friends", "friend group", "relatives", "cousins")] [pick("behind", "behind in safety", "behind secretly", "behind regrettably")], you decided to join the ranks of a Nanotrasen private military contracting group.</b>")
+	to_chat(H, "<B>After leaving your [pick(75;"distant", 20;"close", 5;"ever-lovingly close")] [pick("family", "friends", "band of friends", "friend group", "relatives", "cousins")] [pick("behind", "behind in safety", "behind secretly", "behind regrettably")], you decided to join the ranks of a private military contracting group wokring for Nanotrasen.</b>")
```
Comment (TiviPlus) [inline code/datums/emergency_calls/pmc.dm:8]: wokring

Comment (Pariah919) [reply code/datums/emergency_calls/pmc.dm:8]: imagine being able to read

**Review by TiviPlus — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-67). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**